lib/wallet/client.js

```diff
@@ -77,6 +77,10 @@ class WalletClient extends NodeClient {
 
     return super.rescan(start);
   }
+
```

Comment:
Interesting how the `async` is used in some cases and not in other cases. I think you should make this method `async` because explicit is better than implicit.

https://github.com/bcoin-org/bclient/blob/master/lib/node.js#L61